client/lib/plans/features-list.js

```diff
@@ -532,9 +533,13 @@ export const FEATURES_LIST = {
 		getSlug: () => constants.FEATURE_WORDADS_INSTANT,
 		getTitle: () => i18n.translate( 'Site monetization' ),
 		getDescription: () =>
-			i18n.translate(
-				'Earn money on your site by displaying ads and collecting recurring payments or donations.'
-			),
+			isEnabled( 'earn/rename-payment-blocks' )
+				? i18n.translate(
+						'Earn money on your site by displaying ads and collecting payments or donations.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.